src/main/java/org/cyclops/integrateddynamics/core/evaluate/operator/Operators.java

```diff
@@ -3341,6 +3341,86 @@ public NBTTagCompound getOutput(Triple<NBTTagCompound, String, OperatorBase.Safe
                 return ValueObjectTypeRecipe.ValueRecipe.of(null);
             }).build());
 
+    /**
+     * ------------------------------------ PARSE OPERATORS ------------------------------------
+     */
+
+    /**
+     * Boolean Parse operator which takes a string of form `/(F(alse)?|[+-]?(0x|#)?0+|)/i`.
+     */
+    public static final IOperator PARSE_BOOLEAN = Operators.REGISTRY.register(new ParseOperator<>(ValueTypes.BOOLEAN, v -> {
+      ValueTypeString.ValueString value = v.getValue(0);
+      java.util.regex.Pattern p = java.util.regex.Pattern.compile("\\A(F(alse)?|[+-]?(0x|#)?0+|)\\z", java.util.regex.Pattern.CASE_INSENSITIVE);
```

Comment:
Is it possible to use re2j's `Pattern` instead of Java's `Pattern`? 
Regexes are handled a bit safer there. Probably not needed here, but would be good for consistency.